geotools-suite-8.4/pom.xml

```diff
@@ -78,7 +78,7 @@
 		<dependency>
 			<groupId>org.geotools</groupId>
 			<artifactId>gt-epsg-hsql</artifactId>
-			<version>${bundle.version}</version>
+			<version>13.1</version>
```

Comment:
I originally attempted to update all of geotools to 13.1. I spent a lot of time trying to get it to work without success. hsql is only used once in spatial and as far as I can tell is compatible with 8.4. Maybe we should make a ticket for updating geotools to be sure.